package.json

```diff
@@ -4,7 +4,7 @@
   "description": "markdown-it plugin for embedding JSX syntax instead of raw HTML.",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha test/test.js"
```

Comment:
As a contributor
When I type `npm test`
Then I expect the whole test-suite to run

But mocha said "no tests specified" (or s.th.) like that

This change ensures all test are executed when i run `npm test`